_includes/v22.2/ui/active-statement-executions.md

```diff
@@ -0,0 +1,38 @@
+
+## Active Executions table
+
+Click <img src="{{ 'images/common/ui-columns-button.png' | relative_url }}" alt="Column selector" /> to select the columns to display in the table.
+
+Column | Description
+-----|------------
+Statement Execution ID | The execution ID of the statement execution.
+Statement Execution | The SQL statement that was executed.
+Status | The status of the execution: Preparing, Waiting, or Executing.
+Start Time | The time the execution started.
+Time Spent Waiting | The time the execution spent waiting and experiencing lock contention.
+Application | The name specified by the [`application_name`](show-vars.html#supported-variables) session setting.
+
+To view details of an active statement execution, click an execution ID in the **Statement Execution ID** column to open the **Statement Execution** details page.
+
+## Statement Execution details page
+
+The details page provides the following details on the statement execution.
+
+- **Start Time (UTC)**: the time the execution started.
```

Comment:
I believe all the words following the colons in this list should be capitalized, unless our style specifies otherwise (in which case our docs may be inconsistent).

Reply:
Fixed.